[2024-09-29T12:36:07.766+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-29T12:36:07.793+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_hooks_v01.postgres_to_s3 scheduled__2024-09-20T00:00:00+00:00 [queued]>
[2024-09-29T12:36:07.815+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_hooks_v01.postgres_to_s3 scheduled__2024-09-20T00:00:00+00:00 [queued]>
[2024-09-29T12:36:07.820+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-09-29T12:36:07.857+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): postgres_to_s3> on 2024-09-20 00:00:00+00:00
[2024-09-29T12:36:07.871+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=19786) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-29T12:36:07.872+0000] {standard_task_runner.py:72} INFO - Started process 19789 to run task
[2024-09-29T12:36:07.873+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_hooks_v01', 'postgres_to_s3', 'scheduled__2024-09-20T00:00:00+00:00', '--job-id', '856', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_hooks.py', '--cfg-path', '/tmp/tmp58hg64w_']
[2024-09-29T12:36:07.879+0000] {standard_task_runner.py:105} INFO - Job 856: Subtask postgres_to_s3
[2024-09-29T12:36:07.910+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-09-29T12:36:07.991+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_postgres_hooks_v01.postgres_to_s3 scheduled__2024-09-20T00:00:00+00:00 [running]> on host 6851fee35b72
[2024-09-29T12:36:07.997+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-09-29T12:36:08.219+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='dag_with_postgres_hooks_v01' AIRFLOW_CTX_TASK_ID='postgres_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2024-09-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-20T00:00:00+00:00'
[2024-09-29T12:36:08.220+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-29T12:36:08.235+0000] {base.py:84} INFO - Retrieving connection 'postgres_localhost'
[2024-09-29T12:36:08.248+0000] {dag_with_postgres_hooks.py:29} INFO - Saved orders dat in text file get_rders.txt
[2024-09-29T12:36:08.249+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-29T12:36:08.260+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-29T12:36:08.264+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_hooks_v01, task_id=postgres_to_s3, run_id=scheduled__2024-09-20T00:00:00+00:00, execution_date=20240920T000000, start_date=20240929T123607, end_date=20240929T123608
[2024-09-29T12:36:08.326+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-29T12:36:08.379+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-29T12:36:08.389+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2024-09-29T12:41:17.102+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-29T12:41:17.115+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_hooks_v01.postgres_to_s3 scheduled__2024-09-20T00:00:00+00:00 [queued]>
[2024-09-29T12:41:17.124+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_hooks_v01.postgres_to_s3 scheduled__2024-09-20T00:00:00+00:00 [queued]>
[2024-09-29T12:41:17.126+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-09-29T12:41:17.146+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): postgres_to_s3> on 2024-09-20 00:00:00+00:00
[2024-09-29T12:41:17.156+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=21182) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-29T12:41:17.158+0000] {standard_task_runner.py:72} INFO - Started process 21186 to run task
[2024-09-29T12:41:17.157+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_hooks_v01', 'postgres_to_s3', 'scheduled__2024-09-20T00:00:00+00:00', '--job-id', '885', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_hooks.py', '--cfg-path', '/tmp/tmpjrmqom9m']
[2024-09-29T12:41:17.158+0000] {standard_task_runner.py:105} INFO - Job 885: Subtask postgres_to_s3
[2024-09-29T12:41:17.174+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-09-29T12:41:17.201+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_postgres_hooks_v01.postgres_to_s3 scheduled__2024-09-20T00:00:00+00:00 [running]> on host 6851fee35b72
[2024-09-29T12:41:17.202+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-09-29T12:41:17.273+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='dag_with_postgres_hooks_v01' AIRFLOW_CTX_TASK_ID='postgres_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2024-09-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-20T00:00:00+00:00'
[2024-09-29T12:41:17.274+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-29T12:41:17.290+0000] {base.py:84} INFO - Retrieving connection 'postgres_localhost'
[2024-09-29T12:41:17.299+0000] {dag_with_postgres_hooks.py:29} INFO - Saved orders dat in text file get_rders.txt
[2024-09-29T12:41:17.300+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-29T12:41:17.308+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-29T12:41:17.308+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_hooks_v01, task_id=postgres_to_s3, run_id=scheduled__2024-09-20T00:00:00+00:00, execution_date=20240920T000000, start_date=20240929T124117, end_date=20240929T124117
[2024-09-29T12:41:17.346+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-29T12:41:17.360+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-29T12:41:17.362+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
